config/config.js

```diff
@@ -4,7 +4,7 @@ const _ = require("lodash");
 const fs = require("fs");
 const path = require("path");
 
-const envConfig = process.env.NODE_ENV === "DEVTOOLS_PANEL" ?
+const envConfig = process.env.NODE_ENV === "production" ?
```

Comment:
Thinking this through a little more (see my other comment) I'm not sure what the best solution is but it would be nice to be able to run in prod mode locally.